src/android/io/sentry/SentryCordova.java

```diff
@@ -25,18 +25,26 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
   }
 
   public boolean execute(String action, JSONArray args, final CallbackContext callbackContext) throws JSONException {
-    if(action.equals("install")) {
-      String dsn = args.getString(0);
-
-      SentryAndroid.init(this.cordova.getActivity().getApplicationContext(), options -> {
-        options.setDsn(dsn);
-      });
-      // We need to return false here to not create the captureBreadcrumb hook
-      callbackContext.sendPluginResult(new PluginResult(Status.OK, false));
-    } else {
-      callbackContext.sendPluginResult(new PluginResult(Status.ERROR, "not implemented"));
-    }
-    return true;
+      switch(action) {
+        case "startWithOptions":
```

Comment:
We're keeping the iOS name as a standard?
Since sentry-cocoa is the outliner, wouldn't it be better to call it `init` instead?

Reply:
Hmm the name actually comes from the native bridge on RN (I realized that was actually the first thing I was tasked with at Sentry lol https://github.com/getsentry/sentry-react-native/pull/860). So I figured we should keep the naming the same, save for when we do the unified native bridge module for hybrid SDKs.